packages/editor/src/components/parts/task/notification/NotificationPart.tsx

```diff
@@ -0,0 +1,25 @@
+import { Checkbox } from '../../../widgets';
+import { PathCollapsible } from '../../common';
+import { useTaskData } from '../useTaskData';
+import TemplateSelect from './TemplateSelect';
+import { DEFAULT_TEMPLATE } from './useTemplates';
+
+const NotificationPart = () => {
+  const { task, updateNotification } = useTaskData();
+
+  return (
+    <PathCollapsible label='Notification' path='notification' defaultOpen={ task.notification.suppress || task.notification.template != DEFAULT_TEMPLATE }>
```

Comment:
better would be to compare it with the real defaults from the server backend. Then it would be also correct if we would once change the defaults in the backend or if the defaults may not always be the same.